authorization/permissions/main.go

```diff
@@ -0,0 +1,183 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"log"
+	"net/http"
+	"os"
+	"time"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/arm"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/policy"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/to"
+	"github.com/Azure/azure-sdk-for-go/sdk/azidentity"
+	"github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/authorization/armauthorization"
+	"github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/network/armnetwork"
+	"github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/resources/armresources"
+)
+
+var (
+	subscriptionID            string
+	ObjectID                  string
+	location                  = "westus"
+	resourceGroupName         = "sample-resource-group"
+	virtualNetworkName        = "sample-virtual-network"
+	resourceProviderNamespace = "Microsoft.Network"
+	resourceType              = "virtualNetworks"
+)
+
+func main() {
+	subscriptionID = os.Getenv("AZURE_SUBSCRIPTION_ID")
+	if len(subscriptionID) == 0 {
+		log.Fatal("AZURE_SUBSCRIPTION_ID is not set.")
+	}
+
+	ObjectID = os.Getenv("AZURE_OBJECT_ID")
```

Comment:
local variables should start with lower case letters
```suggestion
	objectID = os.Getenv("AZURE_OBJECT_ID")
```